scripts/database_migration/README.md

```diff
@@ -0,0 +1,53 @@
+# Script Manutenzione del DB per ambienti di Test.
+
+__N.B:__ gli script presenti in questa directory non possono essere eseguiti più di uno alla 
+volta e con una sola esecuzione con lo stesso filesystem. Questo perché i nomi dei file 
+temporanei non sono randomizzati. E potrebbero anche coincidere tra script differenti.
+
+
+## Script per il caricamento delle configurazioni
+- Per le tabelle pn-PaperCap, pn-PaperZone, pn-PaperCost, pn-PaperDeliveryDriver, 
+  pn-PaperDeliveryFile, pn-PaperTender, rifarsi al microservizio pn-paper-channel. 
+  Cartella _pn-paper-channel/scripts/aws/migration_ per quanto riguarda le prime due 
+  tabelle mentre per le altre tabelle si usa la funzionalità di BackOffice 
+  _Gestione gare__
+- Le tabelle pn-aggregates, pn-apiKey, pn-paAggregations, non sono ricopiabili e vanno 
+  rigenerate tramite chiamate rest, eventualmente eseguite tramite funzionalità di test 
+  dell'API-GW di AWS. FIXME generare procedura
+- pn-AuditStorage, viene ricostruita alla prima esecuzione di pn-logsaver-be
+- pn-LastPollForFutureActionTable, viene ricostruita alla prima esecuzione di pn-delivery-push
+- pn-OnboardInstitutions, non può essere copiata perchè ambienti diversi hanno identificativi 
+  selfcare differenti. Vedere procedura sottostante.
+- terraform-lock, pn-PnDeliveryPushShedLock, non vanno migrate, vengono popolate in automatico
+  dai software che le usano.
+- pn-SsTipologieDocumenti, pn-SsAnagraficaClient, pn-SmStates utilizzare lo script presente 
+  nel repository _pn-ss_ nel file _scripts/dynamoDBLoad.sh_
+- pn-EcAnagrafica, n-SmStates utilizzare lo script presente 
+  nel repository _pn-ec_ nel file _scripts/dynamoDBLoad.sh_
+
+## Copia tabella pn-OnboardInstitutions con bonifica degli id
+La tabella pn-OnboardInstitutions contiene l'elenco delle PA che hanno attivato PN, il campo 
+_id_ è correlato a SelfCare ed è differente in ogni ambiente, per questo non si può fare 
+una semplice copia. 
+La procedura è composta di due scipt, 
+- _download_OnboardInstitutions.sh_ per preparare un file che andrà modificato per adeguare 
+  i valori del campo _id_, 
```

Comment:
@marco-vit-pagopa verificare che due prodotti sulla stessa istanza di Selfcare, ad es. Selfcare UAT, ricevano lo stesso identificativo a parità di PA.
In altre parole, una PA venga identificata dallo stesso ID durante onboarding su due prodotti differenti, entrambi connessi allo stesso Selfcare.

Reply:
Si, l'abbiamo provato nel passaggio tra i due dev